mongodb/archivetracking.go

```diff
@@ -0,0 +1,165 @@
+package mongodb
```

Comment:
Mongo collection used by the tracker to know whats pending or finalizing things.